site/search/SearchPanel.tsx

```diff
@@ -322,7 +319,6 @@ const SearchResults = (props: SearchResultsProps) => {
                         query
                     ) {
                         logSiteSearchClickToAlgoliaInsights({
```

Comment:
One last thing: Can you adapt the type of `logSiteSearchClickToAlgoliaInsights`, such that it reflects reality again?